From b8ec0ce733369ca924fbead2fa71b5d642af6224 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Tue, 20 Aug 2024 12:04:04 +0300
Subject: [PATCH] [SW-195340] checkpatch: ignore volatile warning

Volatile is used in our code for HW access, and kernel precautions
regarding usage of 'volatile' is not relevant.

Change-Id: Id25d8e2a798cc63c42826ac28d3f0682291769de
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/.checkpatch.conf b/.checkpatch.conf
index 5f7f686..759013e 100644
--- a/.checkpatch.conf
+++ b/.checkpatch.conf
@@ -6,3 +6,4 @@
 --ignore PRINTK_WITHOUT_KERN_LEVEL
 --ignore LONG_LINE
 --ignore PREFER_SECTION
+--ignore VOLATILE
